app/models/project.rb

```diff
@@ -112,20 +112,10 @@ def self.all_projects(user)
     request.results
   end
 
-  def user_has_access?(user:)
-    return true if user.eligible_sysadmin?
-    metadata_model.data_sponsor == user.uid || metadata_model.data_manager == user.uid ||
-    metadata_model.data_user_read_only.include?(user.uid) || metadata_model.data_user_read_write.include?(user.uid)
-  end
-
```

Comment:
Moved to the presenter to use the Mediaflux data (rather than our fields on the Rails database)